Library/Homebrew/gpg.rb

```diff
@@ -5,7 +5,7 @@ def self.find_gpg(executable)
     which_all(executable).detect do |gpg|
       gpg_short_version = Utils.popen_read(gpg, "--version")[/\d\.\d/, 0]
       next unless gpg_short_version
-      Version.create(gpg_short_version.to_s) == Version.create("2.0")
+      Version.create(gpg_short_version.to_s) == Version.create("2.0") || Version.create("2.1")
```

Comment:
I'd stick parentheses around this as the operator precedence is ambiguous. I'd also consider just doing `>= Version.create("2.0")`